reqMgrClient.py

```diff
@@ -399,8 +427,8 @@ def getWorkflowInfo(url, workflow):
     """
```

Comment:
@ticoann When I try to run against testbed, I get "406 Not Acceptable" error.
I used this url: "cmsweb-testbed.cern.ch"
and this workflow: "jbadillo_RunIIWinter15GS-001-T2_CH_CERN_T0Backfill_150527_153801_6417"